src/08_comprehensions.py

```diff
@@ -11,13 +11,15 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
 y = []
+for x in range(1, 6) : y.append(x)
```

Comment:
search "python comprehensions" for the syntax of a list comprehension. it's a little more concise than having your `for` loops the way you do. Try to rewrite your answers using this syntax, let me know if you don't find it.

Reply:
Switched for loops to comprehensions. Please let me know if this is the way you were talking about!